src/Parallel/AcceleratorDevice.cpp

```diff
@@ -9,16 +9,24 @@ void AcceleratorDevice::bindSyclDevice(int deviceId) {
   std::ostringstream info;
 
   try {
+#ifdef __DPCPP_COMPILER
+    syclDevice = sycl::device(sycl::gpu_selector_v);
+#else
```

Comment:
Wouldn't another `#elif` be sufficient here?

Reply:
done